sdk/src/main/java/com/ifountain/opsgenie/client/OpsGenieClientConstants.java

```diff
@@ -147,6 +148,7 @@ interface API {
         String ESCALATION_TIME = "escalationTime";
         String FORWARDED_FROM = "forwardedFrom";
         String CONTACTS = "contacts";
+        String CONTACT_ID = "contactId";

```

Comment:
can this be part of contact API changes PR?

Reply:
I'll do it